app/controllers/story.controller.js

```diff
@@ -268,3 +268,41 @@ exports.findAllChats = async (req, res) => {
     }
 }
 
+
+
+// Find all suggested stories for a user
+exports.findAllSuggested = async (req, res) => {
+
+    // get user id from request
+    const userId = req.params.userId;
+
+    const user = await db.user.findByPk(userId);
+
+    // get user genres
+    const genres = user.genres;
+
+    // get all stories
+    const stories = await Story.findAll();
+
+    // filter stories by genre and publish status
+
+    const suggestedStories = stories.filter((story) => {
+        return genres.includes(story.genre) && story.published;
+    }
+    );
+
+    if (suggestedStories.length === 0) {
+        Story.findAll().then((data) => {
+            res.send(data);
+        }
+        ).catch((err) => {
+            res.status(500).send({
```

Comment:
should always be logging errors and their context to `console` before sending response to frontend